src/configs/index.js

```diff
@@ -7,6 +7,19 @@ const routes = {
   pathToAddNews: '/newsadd'
 };
 
+export const errLocalization = {
```

Comment:
Please, implement `translations` in the same way as it is done on the frontend part. You should have `translations` folder and separate files for translations